WebApplication/Backend/Services/FeedbackService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Model.Blog;
+using System;
+using System.Collections.Generic;
+using WebApplication.Backend.Repositorys;
+
+namespace WebApplication.Backend.Services
```

Comment:
Service class is for business logic so it might not need the same classes as Repository, although it could have some like GetAll, Add etc. for example. It should have more meaningful methods like ApproveFeedback which then calls the Repository's update method or GetAverageRating etc.